outbox-event-publisher/src/main/java/org/jvalue/outboxeventpublisher/ConfigHelper.java

```diff
@@ -0,0 +1,29 @@
+package org.jvalue.outboxeventpublisher;
+
+import io.debezium.config.Configuration;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.util.function.Consumer;
+
+@Slf4j
+public class ConfigHelper {
+  public static Consumer<Configuration.Builder> fromEnvVar(String prefix) {
+    return builder ->
+      System.getenv().forEach((key, value) -> {
+        var normalizedKey = key.replace("_", ".").toLowerCase();
+        if (normalizedKey.startsWith(prefix)) {
+          builder.with(normalizedKey.replace(prefix, ""), value);
```

Comment:
Please use an explanatory variable here. It's hard to see at first glance what is going on here.